### PR TITLE
make cop request timeout a config paramter

### PR DESCRIPTION
**Pull request by SpadeA-Tang**

### CODE REVIEW

**Comment by cfzjywxk** [comment]
There's a [proposal](https://github.com/pingcap/tidb/pull/45093) about configurable timeout that could be referenced, it's not needed to add a seperate configuration in `kv-client`.
For temporary test it's ok to add configurations as it's simple.

**Comment by SpadeA-Tang** [comment]
> There's a [proposal](https://github.com/pingcap/tidb/pull/45093) about configurable timeout that could be referenced, it's not needed to add a seperate configuration in `kv-client`. For temporary test it's ok to add configurations as it's simple.

It's okay.

**Comment by cfzjywxk** [comment]
Use configuration change on the master branch as the configurable timeout proposal is still under discussion.

**Review by cfzjywxk — APPROVED** [review]
It's needed to comment that it could be removed in the future.

**Comment by SpadeA-Tang** [comment]
> It's needed to comment that it could be removed in the future.

Fine.

File: config/client.go [hunk 1]
```diff
@@ -150,6 +153,7 @@ func DefaultTiKVClient() TiKVClient {
 			AdmissionMaxResultMB:  10,
 			AdmissionMinProcessMs: 5,
 		},
+		CoprReqTimeout: 60 * time.Second,
```
Comment (overvenus) [inline config/client.go:156]:
```suggestion
		CoprReqTimeout: tikv.ReadTimeoutMedium,
```

**Review by overvenus — APPROVED** [review]

**Pull request merged**